<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>Widget</class>
  <widget class="QWidget" name="Widget">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>1920</width>
        <height>1080</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>TreeHugger</string>
    </property>
    <property name="styleSheet">
      <string notr="true">
        border: 0px solid #000;
      </string>
    </property>
    <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
        <rect>
          <x>0</x>
          <y>0</y>
          <width>1919</width>
          <height>1080</height>
        </rect>
      </property>
      <layout class="QVBoxLayout" name="GLSpace"/>
    </widget>
      <widget class="QPushButton" name="pushButton_1">
        <property name="geometry">
          <rect>
            <x>100</x>
            <y>300</y>
            <width>150</width>
            <height>150</height>
          </rect>
        </property>
        <property name="text">
          <string/>
        </property>
        <property name="icon">
          <iconset>
            <disabledon>Laub.png</disabledon>
          </iconset>
        </property>
        <property name="iconSize">
          <size>
            <width>120</width>
            <height>120</height>
          </size>
        </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
          <rect>
            <x>100</x>
            <y>450</y>
            <width>150</width>
            <height>150</height>
          </rect>
        </property>
        <property name="text">
          <string/>
        </property>
        <property name="icon">
          <iconset>
            <disabledon>Laub_lang.png</disabledon>
          </iconset>
        </property>
        <property name="iconSize">
          <size>
            <width>120</width>
            <height>120</height>
          </size>
        </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
          <rect>
            <x>100</x>
            <y>600</y>
            <width>150</width>
            <height>150</height>
          </rect>
        </property>
        <property name="text">
          <string/>
        </property>
        <property name="icon">
          <iconset>
            <disabledon>Pinie.png</disabledon>
          </iconset>
        </property>
        <property name="iconSize">
          <size>
            <width>120</width>
            <height>120</height>
          </size>
        </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
          <rect>
            <x>100</x>
            <y>100</y>
            <width>150</width>
            <height>150</height>
          </rect>
        </property>
        <property name="text">
          <string/>
        </property>
        <property name="icon">
          <iconset>
            <disabledon>Wind.png</disabledon>
          </iconset>
        </property>
        <property name="iconSize">
          <size>
            <width>120</width>
            <height>120</height>
          </size>
        </property>
      </widget>
    
    <widget class="QDial" name="dial">
      <property name="geometry">
        <rect>
          <x>100</x>
          <y>900</y>
          <width>150</width>
          <height>150</height>
        </rect>
      </property>
      <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
        <string notr="true">
          border: 0px solid #000;
        </string>
      </property>
      <property name="minimum">
        <number>0</number>
      </property>
      <property name="maximum">
        <number>9</number>
      </property>
      <property name="value">
        <number>0</number>
      </property>
      <property name="sliderPosition">
        <number>0</number>
      </property>
      <property name="tracking">
        <bool>true</bool>
      </property>
      <property name="orientation">
        <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
        <bool>false</bool>
      </property>
      <property name="invertedControls">
        <bool>false</bool>
      </property>
      <property name="wrapping">
        <bool>false</bool>
      </property>
      <property name="notchesVisible">
        <bool>true</bool>
      </property>
    </widget>
  </widget>
  <layoutdefault spacing="0" margin="0"/>
  <resources/>
  <connections/>
</ui>
